jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -47,7 +47,11 @@ def parse_data(app, file_obj, data_type=None, data_label=None):
     #  supports MaNGA and JWST data.
     if isinstance(file_obj, fits.hdu.hdulist.HDUList):
         _parse_hdulist(app, file_obj, file_name=data_label)
-    elif isinstance(file_obj, str) and os.path.exists(file_obj):
```

Comment:
There is no need to check if file exist, `load_data` at app level already does that.